src/TextBoxSearch/components/TextBoxSearchContainer.ts

```diff
@@ -33,20 +34,25 @@ export interface ContainerState {
     targetListView?: ListView;
     targetNode?: HTMLElement;
     validationPassed?: boolean;
+    datePicker?: mxui.widget.DatePicker;
+    searchNode?: HTMLInputElement;
 }
 
 export default class SearchContainer extends Component<ContainerProps, ContainerState> {
     private dataSourceHelper: DataSourceHelper;
-    private navigationHandler: object;
+    private connections: object[];
     private widgetDOM: HTMLElement;
```

Comment:
`widgetDOM` has no initializer and is not definitely assigned in the constructor i suggest you assign it an initial value
or make it optional and same applies to `dataSourceHelper`